### ISSUE TITLE
Luascript: Text color within lcd.drawFilledRectangle in title not inverted

### ISSUE THREAD

**Issue by jogybaer0815** [EdgeTX/edgetx#473]
TX16S nightly release 6eef71d

Text within title with filled rectangle appears not inverted as expected.

code snippets:

 local function drawScreenTitle(title,page, pages)
  lcd.drawFilledRectangle(0, 0, LCD_W, 30, TITLE_BGCOLOR)
  lcd.drawText(1, 5, title, MENU_TITLE_COLOR)
  lcd.drawText(LCD_W-40, 5, page.."/"..pages, MENU_TITLE_COLOR)
end
 
That appears as black (new standard color - previously red) over filled darkblue rectangle :(
my temporary workaround: 

local function drawScreenTitle(title,page, pages)
--  lcd.drawFilledRectangle(0, 0, LCD_W, 30, TITLE_BGCOLOR)
  lcd.drawRectangle(0, 0, LCD_W, 30, TITLE_BGCOLOR)
  lcd.drawText(1, 5, title, MENU_TITLE_COLOR)
  lcd.drawText(LCD_W-40, 5, page.."/"..pages, MENU_TITLE_COLOR)
end

that is readable.
openTX shows inverted text within filled rectangle
greetings

**Comment by pfeerick** [comment]
So is the problem using the theme colours, rather than specifying your own?

**Comment by jogybaer0815** [comment]
I have only removed the pilot picture from theme

**Comment by pfeerick** [comment]
Pilot picture?

**Comment by raphaelcoeffic** [comment]
> Pilot picture?

He probably means the default background.

**Comment by jogybaer0815** [comment]
sorry for that confusion: yes background.png

**Comment by JimB40** [comment]
still valid?
Do we support color contants legacy? I mean fix OTX bugs

**Comment by pfeerick** [comment]
> Do we support color constants legacy? 

Yes, unless we want to ensure a lot of old scripts become incompatible, which is not an option just yet IMO. 

However, checking the behaviour of OTX vs ETX, I think this was fixed in v2.5 also through theme constant changes... The unreadable red on dark blue certainly seems to be gone ;)

OTX:
![image](https://user-images.githubusercontent.com/5500713/141446518-e97f36e1-9c9c-417e-9db9-5066beb599c4.png)

ETX:
![image](https://user-images.githubusercontent.com/5500713/141446601-94d49364-bfa2-4315-a88a-e7fbee00e4c7.png)

```
local page = 1
local pages = 3
local firstRun = 0

local function drawScreenTitle(title,page, pages)
  lcd.drawFilledRectangle(0, 0, LCD_W, 30, TITLE_BGCOLOR)
  lcd.drawText(1, 5, title, MENU_TITLE_COLOR)
  lcd.drawText(LCD_W-40, 5, page.."/"..pages, MENU_TITLE_COLOR)
end

local function run(event)
    if firstRun == 0 then
      firstRun = 1
      lcd.clear() 
      lcd.drawFilledRectangle(0, 0, LCD_W, 30, TITLE_BGCOLOR)
      lcd.drawText(1, 5, "Welcome (Run)", MENU_TITLE_COLOR)
      lcd.drawText(LCD_W-40, 5, page.."/"..pages, MENU_TITLE_COLOR)
    end

    if event == EVT_VIRTUAL_NEXT_PAGE and page < pages then
      page = page + 1
      drawScreenTitle("My Title (Next)",page, pages)
    end

    if event == EVT_VIRTUAL_PREV_PAGE and page > 1 then
      page = page - 1
      drawScreenTitle("My Title (Prev)",page, pages)
    end

    if event == EVT_VIRTUAL_ENTER_LONG or
       event == EVT_ENTER_LONG or
       event == EVT_MENU_LONG then
        -- exit script
        return 2
    end

  return 0
end

return {run=run}
```
